bundler/spec/resolver/platform_spec.rb

```diff
@@ -337,6 +337,14 @@
         should_resolve_as %w[thin-1.2.7-x64-mingw-ucrt]
       end
     end
+
+    if Gem.rubygems_version > Gem::Version.new("3.3.999")
```

Comment:
Could you change this to `>= 3.3.18`? Just to match what we do in other places, keeping the exact version where the feature was introduced.

Reply:
Done